app/assets/stylesheets/redactor/index.scss

```diff
@@ -295,8 +295,8 @@ body .redactor_toolbar li a.redactor_btn_normalscreen span		  { background-posit
 	vertical-align: middle !important;
 }
 #redactor_image_box {
-	height: 270px; 
-	overflow: auto; 
+	height: 270px;
+	overflow: auto;
```

Comment:
Line should be indented with spaces, not tabs<br>Properties should be ordered height, margin-bottom, overflow